CHANGELOG.md

```diff
@@ -79,7 +79,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * [#7068](https://github.com/osmosis-labs/osmosis/pull/7068) Add CI action that check for typo on new PRs (Thanks @codespell)
 * [#7093](https://github.com/osmosis-labs/osmosis/pull/7093),[#7100](https://github.com/osmosis-labs/osmosis/pull/7100) Lower CPU overheads of the Osmosis epoch.
 * [#7106](https://github.com/osmosis-labs/osmosis/pull/7106) Halve the time of log2 calculation (speeds up TWAP code)
-
+* [#7134](https://github.com/osmosis-labs/osmosis/pull/7134) Update Actions labeler and labeler config to v5.0
```

Comment:
TBH, we don't need a changelog for this. There is a "no changelog" label that can be applied to make the CI happy

Reply:
Ah, i reverted it.